### DISCUSSION TITLE
Ok when npm run dev but ko when npm run start

### DISCUSSION THREAD

**Discussion by 1337-TIM** [documenso/documenso#1056] [General]
hi there 
When I do npm run start I've got an error in marketing 
Error: ENOENT: no such file or directory, open '/var/www/doc/apps/marketing/.next/BUILD_ID'

**Comment by Mythie** [comment]
Presumably marketing hasn't been built, if you don't need marketing anyways just run `npm run start -w @documenso/web`

**Reply by 1337-TIM** [reply]
yes , that's right ;) 
thank you so much
